src/citrine/_session.py

```diff
@@ -35,23 +34,13 @@ class Session(requests.Session):
 
     def __init__(self,
                  refresh_token: str = None,
-                 legacy_scheme: Optional[str] = None,
+                 scheme: str = None,
                  host: str = None,
-                 port: Optional[str] = None,
-                 *,
-                 scheme: str = None):
+                 port: Optional[str] = None):
```

Comment:
I'm tempted to enforce keyword args here, but worry about breaking scripts that start with `Citrine(api_key, scheme, host, port)`

Reply:
Given that using it positionally already emitted a warning (see below), we're fine to enforce it as a keyword.

Reply:
Host should be the same thing, as indicated by that deprecation warning.